chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -144,6 +144,7 @@ spec:
             periodSeconds: {{ .Values.scheduler.livenessProbe.periodSeconds }}
             exec:
               command:
+              - /entrypoint
```

Comment:
Any concerns with this breaking compatibility with other images, not the official image?

Reply:
Do we have any other :) ?  I think we already make many assumptions about the image, so one more does not hurt. Maybe add parameter to override the entrypoint if needed?

Reply:
Yeah, that's how I was leaning as well 👍.

(Maybe I'll try using a custom image one of these days and see what those assumptions are)

Reply:
Mainly those documented here:  https://airflow.apache.org/docs/docker-stack/entrypoint.html